client/pages/driver-profile.jsx

```diff
@@ -69,7 +77,36 @@ export default class DriverProfile extends React.Component {
           </form>
         </div>
         </div>
+        <br></br>
```

Comment:
Instead of a `<br>` we use block-level elements so that they stack on top of each other.

Reply:
This is just scaffolding until the other elements are ready for the vehicle profile page. I need to see both views while creating them.